README.md

```diff
@@ -83,61 +88,10 @@ context) properties. Like this:
 ]
 ```
 
-## Screenshots
-
```

Comment:
IMO, this introduction has some value to users to understand the tool. If we don't want to distract in the main doc, we can move it to separated doc and link it here. Completely deleting it may create the gap and user experience regression.

Reply:
I feel like maybe we should move the screenshots to their respective example section:

- this image  https://raw.githubusercontent.com/rhysd/ss/master/github-action-benchmark/main.png to this section https://github.com/benchmark-action/github-action-benchmark?tab=readme-ov-file#charts-on-github-pages-1
- this image I would remove completely: https://raw.githubusercontent.com/rhysd/ss/master/github-action-benchmark/tooltip.png because by the time i am reading this document, I am probably not interested in what a hover action on a github page does :)
- this image https://raw.githubusercontent.com/rhysd/ss/master/github-action-benchmark/alert-comment.png i would move to https://github.com/benchmark-action/github-action-benchmark?tab=readme-ov-file#commit-comment

What do you think @ningziwen ?